test/acceptance/accessibility.py

```diff
@@ -98,10 +98,10 @@ class StaffAreaA11yTest(OpenAssessmentA11yTest):
     """
     Test the accessibility of the staff area.
 
-    This is testing a problem with "self assessment only".
+    This is testing a problem with "staff assessment only".
```

Comment:
As I was writing the a11y test, I realized that this flow becomes a lot simpler if we use `staff-only` instead of `self-only` with override.